py/test/selenium/webdriver/common/element_attribute_tests.py

```diff
@@ -24,53 +24,53 @@
 def test_should_return_null_when_getting_the_value_of_an_attribute_that_is_not_listed(driver, pages):
```

Comment:
I think the tests for `get_attribute` should stay as long as the method is not removed (there is currently no plan as faur as i can tell). New tests / files for `get_dom_attribute` and `get_property` should be added,